index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">My Logo</a>
+        <img id="menu-icon" src="images/Enabled.png" alt="" />
+      </nav>
+    </header>
+
+    <main>
+      <h1>I'm Shimels, Glad to see you!</h1>
+      <p>
+        Full-stack developer with over 3 years of experience in developing web
+        applications. Proficient in both front-end and back-end technologies,
+        including Angular, ASP.NET Core Web API, Node.js, Flask, and SQL.
+        Skilled in troubleshooting and maintaining databases and server-side
+        infrastructure. Adept at analytical thinking and problem-solving.
+        Excellent communication skills and customer focus.
+      </p>
+      <span> Let's Connect</span>
+      <section id="social-links">
+        <a href="#"> <img src="icones/icons8-twitter-50.png" alt="" /></a>
+        <a href="#"> <img src="icones/icons8-linkedin-50.png" alt="" /></a>
+        <a href="#"> <img src="icones/icons8-github-50.png" alt="" /></a>
+      </section>
+    </main>
+  </body>
```

Comment:
- [ ] As stated in the project requirement, all the elements in the Figma design need to be present in your project. Currently, you do not have the appropriate background images. I'll suggest you get the background images from the Figma design and add them to your project. (You can see the circled background images in the Figma screenshot below )

- - FIGMA DESIGN:
![Selection_078](https://github.com/shimels1/portfolio_microverse/assets/107632780/c22bb8c7-7abd-49aa-a0fc-9d33d87490ab)

Reply:
Should I put each piece of image one by one? or should I find one background and put that?

Reply:
- [ ] As suggested by the previous reviewer, kindly add the appropriate background image given in the Figma design. 😃 
![bg](https://github.com/shimels1/portfolio_microverse/assets/99656687/19296b3e-3342-4cbc-958e-c82a1c6d2786)